website/docs/cloud-docs/workspaces/settings/policies.mdx

```diff
@@ -0,0 +1,15 @@
+---
+page_title: Policies - Workspaces - Terraform Cloud
+description: >-
+  Learn how to adjust the policy evaluation experience according to your preferences.
+---
+
+# Policies
+
+Terraform Cloud provides two different experiences for Sentinel policy evaluations. You can adjust your settings to use the experience you prefer. By default, the newest policy evaluation experience is enabled.
+
+# Toggling Between Sentinel Policy Experiences
+
+To toggle between the two Sentinel policy evaluation experiences, click the **Enable the new Sentinel policy experience** toggle under the **Sentinel Experience** heading. Your changes are persisted automatically.
+
+-> **Note:** If you have a run with a Sentinel policy evaluation open in a different tab, you will need to refresh that page in under to see the changes to the policy evaluation experience.
```

Comment:
We try to avoid call outs unless absolutely necessary, so suggesting we add this sentence to the above paragraph:
```suggestion
```